### PR TITLE
Update PoS docs - added new section

### PR DESCRIPTION
**Pull request by EmpieichO**
Update PoS docs - added new section ... + added a doc requested by Sajal, about Rewards & incentves.

### CODE REVIEW

File: docs/pos/how-to/operating/validator-node/rewards.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+description: Learn about the Polygon Network staking incentives.
+keywords:
+  - docs
+  - matic
+  - polygon
+  - rewards
+  - staking
+  - incentives
+image: https://wiki.polygon.technology/img/polygon-logo.png
+---
```
Comment (kmurphypolygon) [inline docs/pos/how-to/operating/validator-node/rewards.md]:
```suggestion
```

File: docs/pos/how-to/operating/validator-node/rewards.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
```
Comment (kmurphypolygon) [inline docs/pos/how-to/operating/validator-node/rewards.md]:
```suggestion
```

File: docs/pos/reference/commands.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
+This guide provides a curated list of common commands and Polygon-specific operations essential for node operators. Whether you're setting up a full node, validator node or troubleshooting, these commands will assist you in managing your Polygon PoS environment effectively.
+
+## Frequently Used Commands for Bor & Heimdall
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
## Frequently used commands for Bor & Heimdall
```

File: docs/pos/reference/commands.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+| **Start Heimdall bridge-server**      | `$ sudo service heimdalld-bridge start`        |
+| **Locate Bor genesis file**           | `$CONFIGPATH/bor/genesis.json`                 |
+| **Start Bor**                         | `sudo service bor start`                       |
+| **Retrieve Heimdall logs**            | `/var/log/matic-logs/`                         |
+| **Check Heimdall logs**               | `tail -f heimdalld.log`                        |
+| **Check Heimdall rest-server logs**   | `tail -f heimdalld-rest-server.log`            |
+| **Check Heimdall bridge logs**        | `tail -f heimdalld-bridge.log`                 |
+| **Check Bor logs**                    | `tail -f bor.log`                              |
+
+## Useful Configuration Commands
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
## Useful configuration commands
```

File: docs/pos/reference/commands.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+
+### Sync Status of Heimdall
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Sync status of Heimdall
```

File: docs/pos/reference/commands.md [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+Use the tabs below to switch between commands for Bor and Heimdall:
+
+<Tabs
+  defaultValue="bor"
+  values={[
+    { label: 'Bor Commands', value: 'bor', },
+    { label: 'Heimdall Commands', value: 'heimdall', },
+  ]
+}>
+<TabItem value="bor">
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Bor
```

File: docs/pos/reference/commands.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+| `eth.syncing.highestBlock - eth.syncing.currentBlock` |  | Compares the current block of your node to the highest block. |
+| `eth.blockNumber` | `curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "eth_blockNumber","params": []}' localhost:8545` | Returns the latest block number processed by the node. |
+| `debug.setHead("0x"+((eth.getBlock('latest').number) - 1000).toString(16))` |  | Rewinds the blockchain to 1000 blocks prior. |
+| `admin.nodeInfo.enode` |  | Retrieves the public enode URL of the node. |
+| `eth.syncing.currentBlock * 100 / eth.syncing.highestBlock` |  | Calculates the remaining percentage for block synchronization. |
+| `eth.getBlock("latest").number` | `curl http://YourIP:8545 -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"bor_getSigners", "params":["0x98b3ea"]}'` | Queries the height of the latest Bor block. |
+|  | `curl http://YourIP:8545 -X POST -H "Content-Type: application/json" --data '{"method":"eth_chainId","params":[],"id":1,"jsonrpc":"2.0"}'` | Retrieves the `chainID`. |
+
+</TabItem>
+<TabItem value="heimdall">
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Heimdall
```

File: docs/pos/reference/commands.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+| `curl localhost:26657/net_info?` | Returns the number of connected peers using `jq .result.n_peers`. |
+| `curl -s localhost:26657/status` | Retrieves Heimdall's current block height using `jq .result.sync_info.latest_block_height`. |
+| `curl localhost:26657/net_info` | Queries the node using its moniker with `grep moniker`. |
+| `curl -s localhost:26657/status` | Checks if Heimdall is in sync using `jq .result.sync_info.catching_up`. |
+| `curl -s localhost:26657/status` | Verifies Heimdall's sync status using `jq .result \| jq .sync_info`. |
+| `heimdalld unsafe-reset-all` | Resets the database in case of issues. |
+| `curl localhost:26657/status` | Provides comprehensive information about Heimdall. |
+
+</TabItem>
+</Tabs>
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
```

File: docs/pos/reference/commands.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+
+## Node Management Commands
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
## Node management commands
```

File: docs/pos/reference/commands.md [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+
+### Sync Status of Heimdall
+
+To check if Heimdall is synced, run:
+
+```bash
+curl http://localhost:26657/status
+```
+
+### Latest Block Height on Heimdall
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Latest block height on Heimdall
```

File: docs/pos/reference/commands.md [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+
+To check the latest block height on Heimdall, run:
+
+```bash
+curl localhost:26657/status
+```
+
+### Latest Block Height on Bor
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Latest block height on Bor
```

File: docs/pos/reference/commands.md [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+
+To check the latest block height on Bor, use:
+
+```bash
+curl http://<your ip>:8545 -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"bor_getSigners", "params":["0x98b3ea"]}'
+```
+
+### Cleanup: Deleting Remnants of Heimdall and Bor
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Cleanup: deleting remnants of Heimdall and Bor
```

File: docs/pos/reference/commands.md [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+**For Binaries:**
+
+```bash
+sudo rm -rf /etc/bor
+sudo rm /etc/heimdall
+```
+
+### Terminate Bor Process
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Terminate Bor process
```

File: docs/pos/reference/commands.md [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+```
+
+**For Binaries:**
+
+```bash
+cd CS-2003/bor
+bash stop.sh
+```
+
+### Retrieve Latest Peer Details
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Retrieve latest peer details
```

File: docs/pos/reference/commands.md [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+
+```bash
+bor attach bor.ipc
+admin.peers.forEach(function(value){
+    console.log(value.enode+',')
+})
+exit
+```
+
+### Stop Heimdall and Bor Services
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Stop Heimdall and Bor services
```

File: docs/pos/reference/commands.md [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+
+**For Linux packages:**
+
+```bash
+sudo service heimdalld stop
+sudo service bor stop
+```
+
+**For Binaries:**
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
**For binaries:**
```

File: docs/pos/reference/commands.md [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+
+```bash
+pkill heimdalld
+pkill heimdalld-bridge
+cd CS-2001/bor
+bash stop.sh
+```
+
+### Remove Heimdall and Bor Directories
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
### Remove Heimdall and Bor directories
```

File: docs/pos/reference/commands.md [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+
+**For Linux packages:**
+
+```bash
+sudo rm -rf /etc/heimdall/*
+sudo rm -rf /etc/bor/*
+```
+
+**For Binaries:**
```
Comment (kmurphypolygon) [inline docs/pos/reference/commands.md]:
```suggestion
**For binaries:**
```

File: docs/pos/reference/contracts/delegation.md [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+
+Note that `MATIC` is a staking token. A delegator needs to have `MATIC` tokens to participate in the delegation.
+
+Initially, a delegator `D` buys tokens from validator `A` specific pool when `1 MATIC per 1 VATIC`.
+
+When a validator gets rewarded with more `MATIC` tokens, new tokens are added to the pool. Let's say with the current pool of `100 MATIC` tokens, `10 MATIC` rewards are added to the pool. But since the total supply of `VATIC` tokens didn't change due to rewards, the exchange rate becomes `1 MATIC per 0.9 VATIC`. Now, delegator `D` gets more `MATIC` for the same shares.
+
+`VATIC`: Validator specific minted validator share tokens (ERC20 tokens)
+
+## Technical Specification
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/delegation.md]:
```suggestion
## Technical specification
```

File: docs/pos/reference/contracts/delegation.md [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+uint256 public activeAmount; // # of tokens delegated which are part of active stake
+```
+
+Exchange rate is calculated as below:
+
+```js
+ExchangeRate = (totalDelegatedPower + delegatorRewardPool) / totalDelegatorShares
+```
+
+## Methods and Variables
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/delegation.md]:
```suggestion
## Methods and variables
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 21]
```diff
@@ -0,0 +1,97 @@
+Here you will find a list of contracts deployed on Polygon together with their initial address, that is, their location on the blockchain. 
+
+<Tabs
+  defaultValue="mainnet"
+  values={[
+    { label: 'Polygon-Mainnet', value: 'mainnet', },
+    { label: 'Mumbai-Testnet', value: 'mumbai', },
+  ]
+}>
+<TabItem value="mumbai">
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
### Mumbai
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 22]
```diff
@@ -0,0 +1,97 @@
...
+
+Parent Chain: Goerli
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
Parent chain: Goerli
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 23]
```diff
@@ -0,0 +1,97 @@
...
+
+[Child Chain: Mumbai]
+
+| Contracts             | Address                                    |
+|-----------------------|--------------------------------------------|
+| ChildChain            | [0x1EDd419627Ef40736ec4f8ceffdE671a30803c5e](https://mumbai.polygonscan.com/address/0x1EDd419627Ef40736ec4f8ceffdE671a30803c5e/) |
+
+</TabItem>
+
+<TabItem value="mainnet">
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
### Mainnet
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 24]
```diff
@@ -0,0 +1,97 @@
...
+
+Parent Chain: Ethereum Mainnet
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
Parent chain: Ethereum mainnet
```

File: docs/pos/reference/contracts/genesis-contracts.md [hunk 25]
```diff
@@ -0,0 +1,97 @@
...
+| DepositManager        | [0xd505C3822C787D51d5C2B1ae9aDB943B2304eB23](https://etherscan.io/address/0xd505C3822C787D51d5C2B1ae9aDB943B2304eB23) |
+| DepositManagerProxy   | [0x401F6c983eA34274ec46f84D70b31C151321188b](https://etherscan.io/address/0x401F6c983eA34274ec46f84D70b31C151321188b) |
+| WithdrawManager       | [0x017C89Ca4Bda3D66cC65E3d20DD95432258201Ca](https://etherscan.io/address/0x017C89Ca4Bda3D66cC65E3d20DD95432258201Ca) |
+| ExitNFT               | [0xDF74156420Bd57ab387B195ed81EcA36F9fABAca](https://etherscan.io/address/0xDF74156420Bd57ab387B195ed81EcA36F9fABAca) |
+| WithdrawManagerProxy  | [0x2A88696e0fFA76bAA1338F2C74497cC013495922](https://etherscan.io/address/0x2A88696e0fFA76bAA1338F2C74497cC013495922) |
+| ERC20Predicate        | [0x158d5fa3ef8e4dda8a5367decf76b94e7effce95](https://etherscan.io/address/0x158d5fa3ef8e4dda8a5367decf76b94e7effce95) |
+| ERC721Predicate       | [0x54150f44c785d412ec262fe895cc3b689c72f49b](https://etherscan.io/address/0x54150f44c785d412ec262fe895cc3b689c72f49b) |
+
+
+**Child Chain: Polygon Mainnet**
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/genesis-contracts.md]:
```suggestion
**Child chain: Polygon mainnet**
```

File: docs/pos/reference/contracts/stakingmanager.md [hunk 26]
```diff
@@ -0,0 +1,205 @@
...
+**Stakers** are divided into **validators**, **delegators**, and **watchers** (for fraud reporting).
+
+[**StakeManager**](https://github.com/maticnetwork/contracts/blob/develop/contracts/staking/stakeManager/StakeManager.sol) is the main contract for handling validator related activities like `checkPoint` signature verification, reward distribution, and stake management. Since the contract is using **NFT ID** as a source of ownership, change of ownership and signer won't affect anything in the system.
+
+!!!tip
+    
+    From one Ethereum address, a **Staker can only be a validator or delegator** (it's just a design choice, no hard reasons).
+
+
+## Validator Admissions / Replacement
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/stakingmanager.md]:
```suggestion
## Validator admissions / replacement
```

File: docs/pos/reference/contracts/stakingmanager.md [hunk 27]
```diff
@@ -0,0 +1,205 @@
...
+
+### Replacement
+PIP4 introduced the concept of showcasing validator performance for community visibility. If a validator is in an unhealthy state for an extended period of time as outlined in PIP4, they are off-boarded from the network. The validator slot is then made available to those coming off of the waitlist.
+
+!!!info
+    
+    Currently, [<ins>Phase 2 of PART C in PIP4</ins>](https://forum.polygon.technology/t/pip-4-validator-performance-management/9956/24) is being implemented. This is where the community decides on validator prospect evaluation criteria. In time, this exercise will produce an application and admissions process.
+
+
+## Methods and Variables
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/stakingmanager.md]:
```suggestion
## Methods and variables
```

File: docs/pos/reference/contracts/stakingmanager.md [hunk 28]
```diff
@@ -0,0 +1,205 @@
...
+- `voteHash` on which all validators sign (BFT ⅔+1 agreement)
+- This function validates only unique sigs and checks for ⅔+1 power has signed on checkpoint root (inclusion in `voteHash` verification in RootChain contract for all data) `currentValidatorSetTotalStake` provides current active stake.
+- Rewards are distributed proportionally to validator's stake. More on rewards in [Rewards Distribution](https://docs.polygon.technology/pos/how-to/operating/validator-node/#reward-distribution)
+<!-- (https://www.notion.so/Rewards-Distribution-127d586c14544beb9ea326fd3bb5d3a2). -->
+
+### isValidator
+
+Checks if a given validator is active validator for the current epoch.
+
+## Timeline Data Structure
```
Comment (kmurphypolygon) [inline docs/pos/reference/contracts/stakingmanager.md]:
```suggestion
## Timeline data structure
```

File: docs/pos/reference/errors.md [hunk 29]
```diff
@@ -0,0 +1,20 @@
+This guide aims to provide solutions for common errors you might encounter while interacting with Polygon PoS. If you come across an error that is not listed here, please consult Polygon Support or community forums for additional help.
+
+## Table of Node Errors
```
Comment (kmurphypolygon) [inline docs/pos/reference/errors.md]:
```suggestion
## Table of node errors
```

File: docs/pos/reference/mapped-tokens.md [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+
+
+<Tabs
+defaultValue="mainnet"
+values={[
+{ label: 'PoS mainnet', value: 'mainnet', },
+{ label: 'Mumbai testnet', value: 'mumbai', },
+]
+}>
+<TabItem value="mumbai">
```
Comment (kmurphypolygon) [inline docs/pos/reference/mapped-tokens.md]:
```suggestion
### Mumbai
```

File: docs/pos/reference/mapped-tokens.md [hunk 31]
```diff
@@ -0,0 +1,50 @@
...
+| PoS\-WETH         | PoS    | [0x60D4dB9b534EF9260a88b0BED6c486fe13E604Fc](https://goerli.etherscan.io/address/0x60D4dB9b534EF9260a88b0BED6c486fe13E604Fc) | [0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa](https://mumbai.polygonscan.com/address/0xA6FA4fB5f76172d178d61B04b0ecd319C5d1C0aa) |
+| DummyERC20Token   | PoS    | [0x655F2166b0709cd575202630952D71E2bB0d61Af](https://goerli.etherscan.io/address/0x655F2166b0709cd575202630952D71E2bB0d61Af) | [0xfe4F5145f6e09952a5ba9e956ED0C25e3Fa4c7F1](https://mumbai.polygonscan.com/address/0xfe4F5145f6e09952a5ba9e956ED0C25e3Fa4c7F1) |
+| DummyERC721Token  | PoS    | [0x084297B12F204Adb74c689be08302FA3f12dB8A7](https://goerli.etherscan.io/address/0x084297B12F204Adb74c689be08302FA3f12dB8A7) | [0x757b1BD7C12B81b52650463e7753d7f5D0565C0e](https://mumbai.polygonscan.com/address/0x757b1BD7C12B81b52650463e7753d7f5D0565C0e) |
+| DummyERC1155Token | PoS    | [0x2e3Ef7931F2d0e4a7da3dea950FF3F19269d9063](https://goerli.etherscan.io/address/0x2e3Ef7931F2d0e4a7da3dea950FF3F19269d9063) | [0xA07e45A987F19E25176c877d98388878622623FA](https://mumbai.polygonscan.com/address/0xA07e45A987F19E25176c877d98388878622623FA) |
+
+</TabItem>
+<TabItem value="mainnet">
```
Comment (kmurphypolygon) [inline docs/pos/reference/mapped-tokens.md]:
```suggestion
### Mainnet
```

File: docs/pos/reference/mapped-tokens.md [hunk 32]
```diff
@@ -0,0 +1,50 @@
...
+
+| Token Name        | Bridge | Parent Chain Address: Ethereum Mainnet                                                                                | Child Chain: Polygon Network                                                                                                      |
```
Comment (kmurphypolygon) [inline docs/pos/reference/mapped-tokens.md]:
```suggestion
| Token name        | Bridge | Parent chain address: Ethereum mainnet                                                                                | Child chain: Polygon network                                                                                                      |

```

File: docs/pos/reference/mapped-tokens.md [hunk 33]
```diff
@@ -0,0 +1,50 @@
...
+| DummyERC20Token   | PoS    | [0xf2F3bD7Ca5746C5fac518f67D1BE87805a2Be82A](https://etherscan.io/address/0xf2F3bD7Ca5746C5fac518f67D1BE87805a2Be82A) | [0xeFfdCB49C2D0EF813764B709Ca3c6fe71f230E3e](https://polygonscan.com/address/0xeFfdCB49C2D0EF813764B709Ca3c6fe71f230E3e) |
+| DummyERC721Token  | PoS    | [0x71B821aa52a49F32EEd535fCA6Eb5aa130085978](https://etherscan.io/address/0x71B821aa52a49F32EEd535fCA6Eb5aa130085978) | [0x6EBEAC13f6403D19C95b6B75008B12fd21a93Aab](https://polygonscan.com/address/0x6EBEAC13f6403D19C95b6B75008B12fd21a93Aab) |
+| DummyERC1155Token | PoS    | [0x556f501CF8a43216Df5bc9cC57Eb04D4FFAA9e6D](https://etherscan.io/address/0x556f501CF8a43216Df5bc9cC57Eb04D4FFAA9e6D) | [0xA0c68C638235ee32657e8f720a23ceC1bFc77C77](https://polygonscan.com/address/0xA0c68C638235ee32657e8f720a23ceC1bFc77C77) |
+| WDEV | PoS    | [0x4a5df63b0c37b38515e4ee51baf40edd420bf7d5](https://etherscan.io/address/0x4a5df63b0c37b38515e4ee51baf40edd420bf7d5) | [0xa5577d1cec2583058a6bd6d5deac44797c205701](https://polygonscan.com/address/0xa5577d1cec2583058a6bd6d5deac44797c205701) |
+
+</TabItem>
+</Tabs>
```
Comment (kmurphypolygon) [inline docs/pos/reference/mapped-tokens.md]:
```suggestion
```

File: docs/pos/reference/port-management.md [hunk 34]
```diff
@@ -0,0 +1,27 @@
+This guide provides a detailed list of default ports used across Polygon nodes, including Bor and Heimdall. Understanding these ports is crucial for network configuration and effective communication between nodes.
+
+## Bor Node
```
Comment (kmurphypolygon) [inline docs/pos/reference/port-management.md]:
```suggestion
## Bor node
```

File: docs/pos/reference/port-management.md [hunk 35]
```diff
@@ -0,0 +1,27 @@
...
+| WebSocket Server        | 8546  | Can be Public, Internal   | WebSocket port for real-time updates.                                                                           |
+| GraphQL Server          | 8547  | Internal                  | GraphQL port for querying data.                                                                                 |
+| Prometheus Server       | 9091  | Can be Public, Monitoring | Prometheus APIs for Grafana data source. Can be mapped to ports 80/443 via an Nginx reverse proxy.              |
+| Grafana Server          | 3001  | Can be Public, Monitoring | Grafana web server. Can be mapped to ports 80/443 via an Nginx reverse proxy.                                   |
+| Pprof Server            | 7071  | Internal, Monitoring      | Pprof server for collecting Bor metrics.                                                                        |
+| UDP Discovery           | 30301 | Can be Public, Internal   | Default port for Bootnode peer discovery.                                                                       |
+
+## Heimdall Node
```
Comment (kmurphypolygon) [inline docs/pos/reference/port-management.md]:
```suggestion
## Heimdall node
```

File: docs/pos/reference/rpc-endpoints.md [hunk 36]
```diff
@@ -0,0 +1,104 @@
+This guide provides an index of network details for the Polygon Mumbai Testnet and Polygon PoS Mainnet, including their associated RPC and node endpoints.
+
+## Network Details
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
## Network details
```

File: docs/pos/reference/rpc-endpoints.md [hunk 37]
```diff
@@ -0,0 +1,104 @@
...
+
+<Tabs
+  defaultValue="mainnet"
+  values={[
+    { label: 'PoS Mainnet', value: 'mainnet' },
+    { label: 'Mumbai Testnet', value: 'mumbai' },
+  ]}
+>
+
+<TabItem value="mumbai">
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
### Mumbai
```

File: docs/pos/reference/rpc-endpoints.md [hunk 38]
```diff
@@ -0,0 +1,104 @@
...
+
+The Mumbai Testnet serves as a replica of the Polygon Mainnet and is primarily used for testing. Obtain testnet tokens from the [faucet](https://faucet.polygon.technology/). Note that these tokens hold no value and differ from MATIC.
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
The Mumbai testnet serves as a replica of the Polygon mainnet and is primarily used for testing. Obtain testnet tokens from the [faucet](https://faucet.polygon.technology/). Note that these tokens hold no value and differ from MATIC.
```

File: docs/pos/reference/rpc-endpoints.md [hunk 39]
```diff
@@ -0,0 +1,104 @@
...
+
+| Properties       | Network Details                                                                                     |
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
| Properties       | Network details                                                                                     |
```

File: docs/pos/reference/rpc-endpoints.md [hunk 40]
```diff
@@ -0,0 +1,104 @@
...
+| Chain ID         | `80001`                                                                                            |
+| Gas Token        | MATIC                                                                                  |
+| Gas Station      | [Mumbai Gas Station](https://gasstation-mumbai.matic.today/v2)                                       |
+| RPC Endpoint     | [https://rpc-mumbai.matic.today](https://rpc-mumbai.matic.today)                                     |
+| Node Endpoint    | [wss://rpc-mumbai.matic.today](wss://rpc-mumbai.matic.today)                                        |
+| Heimdall API     | [https://heimdall-api-testnet.polygon.technology](https://heimdall-api-testnet.polygon.technology)   |
+| Block Explorer   | [https://mumbai.polygonscan.com/](https://mumbai.polygonscan.com/)                                   |
+
+!!!note
+    Additional Information
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
    Additional information
```

File: docs/pos/reference/rpc-endpoints.md [hunk 41]
```diff
@@ -0,0 +1,104 @@
...
+| Network Name     | **Mumbai**                                                                                          |
+| Parent Chain     | **[Goërli](https://goerli.net/)**                                                                   |
+| Chain ID         | `80001`                                                                                            |
+| Gas Token        | MATIC                                                                                  |
+| Gas Station      | [Mumbai Gas Station](https://gasstation-mumbai.matic.today/v2)                                       |
+| RPC Endpoint     | [https://rpc-mumbai.matic.today](https://rpc-mumbai.matic.today)                                     |
+| Node Endpoint    | [wss://rpc-mumbai.matic.today](wss://rpc-mumbai.matic.today)                                        |
+| Heimdall API     | [https://heimdall-api-testnet.polygon.technology](https://heimdall-api-testnet.polygon.technology)   |
+| Block Explorer   | [https://mumbai.polygonscan.com/](https://mumbai.polygonscan.com/)                                   |
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
| Network name     | **Mumbai**                                                                                          |
| Parent chain     | **[Goërli](https://goerli.net/)**                                                                   |
| Chain ID         | `80001`                                                                                            |
| Gas token        | MATIC                                                                                  |
| Gas station      | [Mumbai gas station](https://gasstation-mumbai.matic.today/v2)                                       |
| RPC endpoint     | [https://rpc-mumbai.matic.today](https://rpc-mumbai.matic.today)                                     |
| Node endpoint    | [wss://rpc-mumbai.matic.today](wss://rpc-mumbai.matic.today)                                        |
| Heimdall API     | [https://heimdall-api-testnet.polygon.technology](https://heimdall-api-testnet.polygon.technology)   |
| Block Explorer   | [https://mumbai.polygonscan.com/](https://mumbai.polygonscan.com/)                                   |
```

File: docs/pos/reference/rpc-endpoints.md [hunk 42]
```diff
@@ -0,0 +1,104 @@
...
+
+!!!note
+    Additional Information
+
+    For more details, refer to this [**JSON data**](https://static.polygon.technology/network/testnet/mumbai/index.json).
+
+
+</TabItem>
+
+<TabItem value="mainnet">
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
### Mainnet
```

File: docs/pos/reference/rpc-endpoints.md [hunk 43]
```diff
@@ -0,0 +1,104 @@
...
+| Network Name     | **Polygon**                                                                                         |
+| Parent Chain     | **Ethereum**                                                                                        |
+| Chain ID         | `137`                                                                                               |
+| Gas Token        | MATIC                                                                                               |
+| Gas Station      | [PolygonScan Gas Tracker](https://polygonscan.com/gastracker)                                        |
+| RPC Endpoint     | [https://polygon-rpc.com/](https://polygon-rpc.com/)                                                |
+| Node Endpoint    | [wss://rpc-mainnet.matic.network](wss://rpc-mainnet.matic.network)                                  |
+| Heimdall API     | [https://heimdall-api.polygon.technology](https://heimdall-api.polygon.technology)                   |
+| Block Explorer   | [https://polygonscan.com/](https://polygonscan.com/)                                                 |
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
The native token for the Polygon PoS mainnet is MATIC, which is used for transaction fees.

| Properties       | Network details                                                                                     |
| ---------------- | -------------------------------------------------------------------------------------------------- |
| Network name     | **Polygon**                                                                                         |
| Parent chain     | **Ethereum**                                                                                        |
| Chain ID         | `137`                                                                                               |
| Gas token        | MATIC                                                                                               |
| Gas station      | [PolygonScan Gas Tracker](https://polygonscan.com/gastracker)                                        |
| RPC endpoint     | [https://polygon-rpc.com/](https://polygon-rpc.com/)                                                |
| Node endpoint    | [wss://rpc-mainnet.matic.network](wss://rpc-mainnet.matic.network)                                  |
| Heimdall API     | [https://heimdall-api.polygon.technology](https://heimdall-api.polygon.technology)                   |
| Block explorer   | [https://polygonscan.com/](https://polygonscan.com/)                                                 |
```

File: docs/pos/reference/rpc-endpoints.md [hunk 44]
```diff
@@ -0,0 +1,104 @@
...
+
+!!!note
+    Additional Information
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
    Additional information
```

File: docs/pos/reference/rpc-endpoints.md [hunk 45]
```diff
@@ -0,0 +1,104 @@
...
+
+    For more details, refer to this [**JSON data**](https://github.com/maticnetwork/static/blob/master/network/mainnet/v1/index.json).
+
+
+</TabItem>
+</Tabs>
+
+## RPC API Methods
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
## RPC API methods
```

File: docs/pos/reference/rpc-endpoints.md [hunk 46]
```diff
@@ -0,0 +1,104 @@
...
+
+Developers can interact with on-chain data and execute various types of transactions using network endpoints. These APIs adhere to the JSON-RPC standard, a stateless, lightweight remote procedure call (RPC) protocol.
+
+!!!info
+    Getting Started with RPC Calls
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
    Getting started with RPC calls
```

File: docs/pos/reference/rpc-endpoints.md [hunk 47]
```diff
@@ -0,0 +1,104 @@
...
+
+    For a comprehensive list of API documentation, visit [**Polygon JSON-RPC calls**](https://edge-docs.polygon.technology/docs/get-started/json-rpc-commands/).
+
+    To explore API requests without any setup, fix failing requests, or discover new methods on the Polygon network, try the [**Composer App**](https://composer.alchemyapi.io).
+
+
+### Infrastructure Providers
```
Comment (kmurphypolygon) [inline docs/pos/reference/rpc-endpoints.md]:
```suggestion
### Infrastructure providers
```

File: docs/pos/reference/snapshot-instructions-heimdall-bor.md [hunk 48]
```diff
@@ -0,0 +1,248 @@
+When setting up a new sentry, validator, or full node server, it is recommended that you use snapshots for faster syncing without having to sync over the network. Using snapshots will save you several days for both Heimdall and Bor. **Note: We no longer support bor archive snapshots due to unsustainable data growth.** 
+
+!!!tip
+    
+    For the latest snapshot, please visit [<ins>Polygon Chains Snapshots</ins>](https://snapshot.polygon.technology/).
+
+
+## Client Snapshots
```
Comment (kmurphypolygon) [inline docs/pos/reference/snapshot-instructions-heimdall-bor.md]:
```suggestion
## Client snapshots
```

**Review by kmurphypolygon — APPROVED** [review]

**Review by EmpieichO — COMMENTED** [review]
Changes accepted.

**Pull request merged**